core/trino-main/src/main/java/io/trino/execution/scheduler/faulttolerant/EventDrivenFaultTolerantQueryScheduler.java

```diff
@@ -1255,7 +1258,7 @@ public boolean isSpeculative()
         private IsReadyForExecutionResult isReadyForExecution(SubPlan subPlan)
         {
             boolean standardTasksInQueue = schedulingQueue.getTaskCount(STANDARD) > 0;
-            boolean standardTasksWaitingForNode = preSchedulingTaskContexts.getWaitingForNodeTasksCount(STANDARD) > 0;
+            boolean standardTasksWaitingForNode = preSchedulingTaskContexts.getWaitingForNodeTotalCount(STANDARD) > 0;
```

Comment:
why the method name change?

Reply:
Because we split the method into two. Total tasks waiting for node for given execution class, and another method which only counts tasks which wait for node from list given in parameter.
See how it looks with current naming.